creator-node/src/services/stateMachineManager/stateReconciliation/SecondarySyncHealthTrackerConstants.ts

```diff
@@ -0,0 +1,19 @@
+// See prometheus.constants.js for reference on potential errors
```

Comment:
maybe valuable to denote either in comment or in variable name that these are NUMBER_OF_DAILY_RETRIES